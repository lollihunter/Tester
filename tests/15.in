20 20 20
1 1
6 13
3 15
9 14
14 6
10 17
6 19
15 19
7 15
19 18
18 16
9 16
14 5
7 3
19 20
6 9
11 9
19 7
2 3
20 20
