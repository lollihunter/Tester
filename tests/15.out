10
1 19 16 2 12 6 8 11 10 20 
